feat: patch to v1.8.0

sdjeong-zenaad opened the pull request:
update proto for xpla 1.8

sdjeong-zenaad committed:
feat: patch to v1.8.0

nowooj commented on .gitmodules:
```diff
@@ -7,15 +7,9 @@
 	url = https://github.com/xpladev/xpla.git
-[submodule "ethermint"]
-	path = ethermint
-	url = https://github.com/xpladev/ethermint.git
```
It would be nice to keep ethermint for backwards compatibility (pre-upgrade tx), etc. If not, it would be a good idea to create legacy/ethermint/proto in xpladev/xpla.

sdjeong-zenaad replied:
fixed

sdjeong-zenaad committed:
feat: patch to v1.8.0

sdjeong-zenaad commented:
Added `/legacy/ethermint/proto` from the `xpla` package for backward compatibility.
Also, updated the `cosmos-sdk` and `evm` protos to use the ones from the official cosmos repository instead of the xpladev repository.
